{"trial_id": "01", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "units", "default": null, "conditions": [], "min_value": 32, "max_value": 512, "step": 32, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "dropout", "default": 0.0, "conditions": [], "min_value": 0.0, "max_value": 0.5, "step": 0.1, "sampling": "linear"}}], "values": {"units": 416, "dropout": 0.0}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n    results = self.hypermodel.fit(hp, model, *args, **kwargs)\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n    return model.fit(*args, **kwargs)\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 123, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/tensorflow/python/eager/execute.py\", line 53, in quick_execute\n    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,\ntensorflow.python.framework.errors_impl.InvalidArgumentError: Graph execution error:\n\nDetected at node sequential_1/lstm_1/CudnnRNN defined at (most recent call last):\n  File \"/N/soft/sles15/deeplearning/Python-3.10.10/lib/python3.10/runpy.py\", line 196, in _run_module_as_main\n\n  File \"/N/soft/sles15/deeplearning/Python-3.10.10/lib/python3.10/runpy.py\", line 86, in _run_code\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel_launcher.py\", line 18, in <module>\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/traitlets/config/application.py\", line 1075, in launch_instance\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/kernelapp.py\", line 739, in start\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/tornado/platform/asyncio.py\", line 205, in start\n\n  File \"/N/soft/sles15/deeplearning/Python-3.10.10/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n\n  File \"/N/soft/sles15/deeplearning/Python-3.10.10/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n\n  File \"/N/soft/sles15/deeplearning/Python-3.10.10/lib/python3.10/asyncio/events.py\", line 80, in _run\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 545, in dispatch_queue\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 534, in process_one\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 437, in dispatch_shell\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/ipkernel.py\", line 359, in execute_request\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 778, in execute_request\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/ipkernel.py\", line 446, in do_execute\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/ipykernel/zmqshell.py\", line 549, in run_cell\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3075, in run_cell\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3130, in _run_cell\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/IPython/core/async_helpers.py\", line 129, in _pseudo_sync_runner\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3334, in run_cell_async\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3517, in run_ast_nodes\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3577, in run_code\n\n  File \"/tmp/ipykernel_114307/832800864.py\", line 29, in <module>\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/base_tuner.py\", line 234, in search\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 118, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/backend/tensorflow/trainer.py\", line 323, in fit\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/backend/tensorflow/trainer.py\", line 117, in one_step_on_iterator\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/backend/tensorflow/trainer.py\", line 105, in one_step_on_data\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/backend/tensorflow/trainer.py\", line 56, in train_step\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 118, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/layers/layer.py\", line 816, in __call__\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 118, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/ops/operation.py\", line 42, in __call__\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 157, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/models/sequential.py\", line 203, in call\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/models/functional.py\", line 188, in call\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/ops/function.py\", line 153, in _run_through_graph\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/models/functional.py\", line 572, in call\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 118, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/layers/layer.py\", line 816, in __call__\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 118, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/ops/operation.py\", line 42, in __call__\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 157, in error_handler\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/layers/rnn/lstm.py\", line 538, in call\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/layers/rnn/rnn.py\", line 397, in call\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/layers/rnn/lstm.py\", line 512, in inner_loop\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/backend/tensorflow/rnn.py\", line 822, in lstm\n\n  File \"/N/u/rajaali/BigRed200/.local/lib/python3.10/site-packages/keras/src/backend/tensorflow/rnn.py\", line 933, in _cudnn_lstm\n\nDnn is not supported\n\t [[{{node sequential_1/lstm_1/CudnnRNN}}]] [Op:__inference_one_step_on_iterator_2639]\n"}